### DISCUSSION TITLE
Concatenation operator - spaces being trimmed

### DISCUSSION THREAD

**Discussion by erikas-tranauskas** [PHP-CS-Fixer/PHP-CS-Fixer#6944] [Q&A] [answered]
If you use the concatenation operator all the spaces are trimmed.

```
$firstString = 'test';
$secondString = 'test';
$thirdString = $firstString . $secondString;
```
The result after using the fixer is this:
`$thirdString = $firstString.$secondString;`

Any way to fix this? 

I use the .php-cs-fixer.dist.php config file which is the following:

```
<?php

$finder = (new PhpCsFixer\Finder())
    ->in(__DIR__)
    ->exclude('var')
;

return (new PhpCsFixer\Config())
    ->setRules([
        '@Symfony' => true,
    ])
    ->setFinder($finder)
;
```

Version I use is v3.14.2

**Comment by Wirone** [comment, marked as answer]
You  should configure [`concat_space`](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/operator/concat_space.rst) to use `one` strategy.
